docs/moduleCom7Regional.rst

```diff
@@ -0,0 +1,316 @@
+   call command below with the avalanche directory as argument
+4. Execute from ``AvaFrame/avaframe`` directory:
+
+.. code-block:: bash
+
+    python runScripts/runSplitInputs.py
+
+---------------
+
+Running multiple avalanche dirs
+===============================
+
+Input
+-----
+A directory structure containing pre-configured avalanche directories (containing an ``Inputs`` folder) is required. For input preparation use
+:ref:`moduleCom7Regional:Regional Input Management`, which splits merged input data into standard :ref:`com1DFA <moduleCom1DFA:com1DFA: DFA-Kernel>`
+inputs across multiple avalanche directories.
+
+Example of a valid directory structure (as produced by the regional input management above)::
+
+    avalancheDir
+    ├── Inputs/       #NOT being used for running; optional
+    └── com7Regional/ #This is the default name, can be changed via .ini setup
+        ├── sub_avalanche1/
+        │   └── Inputs/
+        │       ├── REL/*.shp
+        │       └── *.asc or *.tif
+        ├── sub_avalanche2/
+        │   └── Inputs/
+        │       ├── REL/*.shp
+        │       └── *.asc or *.tif
+        └── ...
+
+
+Output
+------
+Outputs are organized in two levels:
+
+**1. Merged rasters** and **2. Individual outputs (per sub_avalanche directory)**
+
+Merged rasters
+^^^^^^^^^^^^^^
+Configure in ``com7RegionalCfg.ini`` (or local):
+
+.. code-block:: ini
+
+    [GENERAL]
+    mergeOutput = True
+    mergeTypes = pfv # Available options: [ppr|pfv|pft|pta|FT|FV|P|FM|Vx|Vy|Vz|TA]
+    mergeMethod = max  # Available options: [max|min|sum|count]
+
+Produces merged rasters of all peakFile results found within the sub-avalanche directories, for each
+``mergeTypes`` and ``mergeMethod`` configured, in ``<avalancheDir>/com7Regional/mergedRasters/``. The merged
+raster combines output from ALL sub-avalanches.
+
+Creates::
+
+    avalancheDir
+    ├── ....
+    └── com7Regional/
+        ├── sub_avalanche1/
+        ...
+        └── mergedRasters/ <- this one is created
+
+Individual outputs
+^^^^^^^^^^^^^^^^^^
+After running com7 with the given module (e.g. :ref:`com1DFA <com1DFA>`), the standard output is located
+within each of the sub-avalanche directories within e.g. ``<avalancheDir>/com7Regional/<avalancheDir>/Outputs/com1DFA``.
+Additionally, com7 provides the option of aggregating all output peakFiles and tSteps results into a single directory
+for easier management, either through copying or moving the files after an executed run.
+
+Configure in ``com7RegionalCfg.ini`` (or local):
+
+.. code-block:: ini
+
+    [GENERAL]
+    copyPeakFiles = True
+    moveInsteadOfCopy = False
+
+Creates::
+
+    avalancheDir
+    ├── ....
+    └── com7Regional/
+        ├── sub_avalanche1/
+        ...
+        └── allPeakFiles/ <- this one is created
+
+Configuration
+-------------
+Three configuration files are used (locals are preferred if they exist):
```

Comment:
not true for com1DFACfg.ini unless in the com7REgionalCfg.ini setDefaultConfig=False in the com1DFA override section